taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from taxi.views import index
```

Comment:
There should be a blank line between the import of `path` from `django.urls` and the import of `index` from `taxi.views` to separate third-party imports from local application imports, as per the checklist.